frontend/src/features/permits/components/permit-list/ApplicationsInProgressList.tsx

```diff
@@ -75,8 +78,17 @@ export const ApplicationsInProgressList = ({ onCountChange }:
     : [],
   });
 
+  const applicationPermitsPendingQuery = useApplicationsInProgressQuery({
+    pendingPermits: true,
+  });
+
   const { data, isError, isPending, isFetching } = applicationsQuery;
 
+  const showPendingBanner = 
+    (applicationPermitsPendingQuery?.data?.items &&
+      applicationPermitsPendingQuery?.data?.items?.length > 0
+    ) ?? false;
```

Comment:
```suggestion
  const showPendingBanner = 
    applicationPermitsPendingQuery?.data?.items?.length > 0;
```

Reply:
reverting this suggestion - eslint hates it